src/components/Feedback.vue

```
@@ -73,14 +73,37 @@
 import { ref } from 'vue'
 import { Toast } from 'bootstrap'
 import { apiUrl } from '@/utils/env'
+import { getCredential } from '@/utils/auth'
```

ESLint

[error] 'getCredential' is defined but never used.

(no-unused-vars)